netmanager/src/config/urls/analytics.js

```diff
@@ -17,12 +17,12 @@ export const EXCEEDANCES_URI = `${BASE_ANALYTICS_URL}/analytics/dashboard/exceed
 export const GET_SITES = `${BASE_ANALYTICS_URL}/analytics/dashboard/sites`;
 
 // pending verification
-export const GET_MONITORING_SITES_LOCATIONS_URI = `${BASE_ANALYTICS_URL}/analytics/dashboard/monitoringsites/locations?organisation_name=KCCA`;
+export const GET_MONITORING_SITES_LOCATIONS_URI = `${BASE_ANALYTICS_URL}/analytics/dashboard/sites/locations?organisation_name=KCCA`;
```

Comment:
thanks @FaithDaka for this amazing work.

I am just curious about the tenant name, why is it hard coded?
Anyway, I am even every sure of how these endpoints work, perhaps I need to do some sort of refresher?